src/app/gallery/gallery.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="g-screen-container flex flex-col gap-5">
```

Comment:
`flex flex-col gap-5` should have been transform into a unique class let say 'gallery-container' or just 'container' for instance. 
And you should do the same for all others element